src/robot/client.ts

```diff
@@ -369,11 +375,11 @@ export class RobotClient extends EventDispatcher implements Robot {
     return this.peerConn?.iceConnectionState === 'connected';
   }
 
-  public async connect(
-    priority?: number,
+  public async connect({
+    priority,
     authEntity = this.savedAuthEntity,
-    creds = this.savedCreds
-  ) {
+    creds = this.savedCreds,
+  }: ConnectOptions) {
```

Comment:
I'd recommend making this object optional so that `this.connect()` could still be called with no args:

```ts
  public async connect({
    priority,
    authEntity = this.savedAuthEntity,
    creds = this.savedCreds,
  }: ConnectOptions = {}) {
```

Reply:
Thanks for that suggestion! Made